js/table/tabpleplacemodel.ts

```diff
@@ -48,10 +48,15 @@ export class TablePlaceModel {
     public BackCards: KnockoutObservableArray<string>;
 
     /*
-     * Actual hand cards which player has in this game.
+     * Visible hand cards which player has in this game.
      */
     public HandCards: KnockoutObservableArray<string>;
 
+    /*
+     * Actual hand cards which player has in this game.
+     */
+    public ActualHandCards: KnockoutObservableArray<string>;
```

Comment:
Instead of creation of *actual* hands which we already have. Would be better create Knockout computed which will be represent `DisplayedHandCards`. It will require change in the template, but since functionality of the `HandCards` does not covered by tests we may forget some rare cases and it is better does not touch it.